airflow/contrib/example_dags/example_gcp_dataproc.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Example Airflow DAG that show how to use various Dataproc
+operators to manage a cluster and submit jobs.
+"""
+
+import os
+import airflow
+from airflow import models
+from airflow.contrib.operators.dataproc_operator import (
+    DataprocClusterCreateOperator,
+    DataprocClusterDeleteOperator,
+    DataprocClusterScaleOperator,
+    DataProcSparkSqlOperator,
+    DataProcSparkOperator,
+    DataProcPySparkOperator,
+    DataProcPigOperator,
+    DataProcHiveOperator,
+    DataProcHadoopOperator,
+)
+
+PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "an-id")
+CLUSTER_NAME = os.environ.get("GCP_DATAPROC_CLUSTER_NAME", "example-project")
+REGION = os.environ.get("GCP_LOCATION", "europe-west1")
+ZONE = os.environ.get("GCP_REGION", "europe-west-1b")
+OUTPUT_BUCKET = os.environ.get("GCP_DATAPROC_OUTPUT_BUCKET", "system-tests-outputs")
+OUTPUT_FOLDER = "wordcount"
+OUTPUT_PATH = "gs://{}/{}/".format(OUTPUT_BUCKET, OUTPUT_FOLDER)
+
+PYSPARK_MAIN = os.environ.get(
+    "PYSPARK_MAIN",
+    "gs://dataproc-examples-2f10d78d114f6aaec76462e3c310f31f/src/pyspark/hello-world/hello-world.py",
```

Comment:
Can you enter a more generic value here? The contents of this bucket should be created when running system tests

Reply:
I see your point but why add additional steps when we can use publicly available resource?

Reply:
For two reasons:
" Because this bucket may disappear at any time. We will lose part of the project in this way.
* it is not a good idea to download something from a random bucket and execute it. it is not safe, because
 its contents may change.

Reply:
So you suggest to have this `hello-world.py` in Apache repo, upload it to bucket and then use the created bucket?

Reply:
Separate files will be confusing.  If they will only be used in one place, they may be included in the code.
Reference: 
https://github.com/apache/airflow/blob/master/tests/gcp/operators/test_cloud_build_system_helper.py#L59-L63
https://github.com/apache/airflow/blob/master/tests/gcp/operators/test_cloud_build_system_helper.py#L51-L55